Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -65,20 +61,19 @@ - (void)commontInit
     OWSAssert(!self.avatarView);
 
     [self setTranslatesAutoresizingMaskIntoConstraints:NO];
-    self.layoutMargins = UIEdgeInsetsZero;
-    self.contentView.layoutMargins = UIEdgeInsetsZero;
-    self.preservesSuperviewLayoutMargins = NO;
-    self.contentView.preservesSuperviewLayoutMargins = NO;
+    self.layoutMargins = UIEdgeInsetsMake(0, self.cellHMargin, 0, self.cellHMargin);
+    self.contentView.preservesSuperviewLayoutMargins = YES;
 
```

Comment:
Don't we still need `self.contentView.layoutMargins = UIEdgeInsetsZero;`?

Reply:
Done!